pkg/db/query/test_queries.go

```diff
@@ -57,6 +58,7 @@ const (
 func TestReportsByVariant(
 	dbc *db.DB,
 	release string,
+	reportType v1.ReportType, // defaults to "current" or last 7 days vs prev 7 days
```

Comment:
We should agree on the proper value for this, we both seem to use different terms - https://issues.redhat.com/browse/TRT-548. I noticed the metrics are now inconsistent since the ones I added recently use `default` as the period.

The UI always called this `default` and sent that to the API.  I tend to prefer `default`, as current is overloaded (the twoDay results are also "current"), but perhaps it should really be called sevenDay.   (The API interprets anything other than `twoDay` as the seven day report, so none of these discrepancies cause bugs yet.)